azure-cs-botservice/Azure.BotService.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
I would delete this file: we use one solution file for all .NET examples

Reply:
Done